tests/utils/common/common.py

```diff
@@ -168,6 +185,9 @@ def _start_qemu(qenv, conn, qemu_wrapper):
     env = dict(os.environ)
     env.update(qenv)
 
+    env["PORT_NUMBER"] = str(8822 + get_worker_index())
+    env["VNC_NUMBER"] = str(23 + get_worker_index())
```

Comment:
Q: A little odd to assign for instance `25`, no?

Reply:
I'm not sure I understand.. It's just arbitrary numbers AFAIK. We just chose 23 because it far away from the default of `0`, which could already be in use on the system (some screen sharing programs use it for example).

Reply:
I was thinking of ports. nvm

Reply:
Ah no, the index is an offset from the VNC port, which I can't remember at the moment, but it's definitely not 23. 🙂